data.json

```diff
@@ -365,4 +365,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/53856673/102192030-10423b00-3e6f-11eb-8494-a684cebb2ecc.png",
     "issueNumber": 186
   }
+    {
```

Comment:
A comma `,` is needed between the json objects and I think the indentation is not quite right